specification/compute/resource-manager/Microsoft.Compute/stable/2020-09-30/examples/GetInformationAboutAManagedDisk.json

```diff
@@ -3,54 +3,46 @@
     "subscriptionId": "{subscription-id}",
     "resourceGroupName": "myResourceGroup",
     "api-version": "2020-09-30",
-    "diskName": "myManagedDisk"
+    "diskName": "myDisk"
   },
   "responses": {
     "200": {
       "body": {
-        "managedBy": "/subscriptions/123caaa-123v-v211-a49f-f88ccac5bf88/resourceGroups/ResourceGroupName/providers/Microsoft.Compute/virtualMachines/TestVM414689371c88843d65ec",
+        "name": "myDisk",
+        "id": "/subscriptions/{subscription-id}/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/mydisk",
+        "type": "Microsoft.Compute/disks",
+        "location": "westus",
         "sku": {
-          "name": "Standard_LRS"
+          "name": "Standard_LRS",
+          "tier": "Standard"
         },
         "properties": {
           "osType": "Windows",
-          "creationData": {
-            "createOption": "Empty"
+          "hyperVGeneration": "V1",
+          "purchasePlan": {
+            "name": "test_sku",
+            "publisher": "test_test_pmc2pc1",
+            "product": "marketplace_vm_test"
           },
-          "diskSizeGB": 10,
-          "encryptionSettingsCollection": {
-            "enabled": true,
-            "encryptionSettings": [
-              {
-                "diskEncryptionKey": {
-                  "sourceVault": {
-                    "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.KeyVault/vaults/myVMVault"
-                  },
-                  "secretUrl": "https://myvmvault.vault-int.azure-int.net/secrets/{secret}"
-                },
-                "keyEncryptionKey": {
-                  "sourceVault": {
-                    "id": "/subscriptions/{subscriptionId}/resourceGroups/myResourceGroup/providers/Microsoft.KeyVault/vaults/myVMVault"
-                  },
-                  "keyUrl": "https://myvmvault.vault-int.azure-int.net/keys/{key}"
-                }
-              }
-            ]
+          "creationData": {
+            "createOption": "FromImage",
+            "imageReference": {
+              "id": "/Subscriptions/{subscription-id}/Providers/Microsoft.Compute/Locations/westus/Publishers/test_test_pmc2pc1/ArtifactTypes/VMImage/Offers/marketplace_vm_test/Skus/test_sku/Versions/1.0.0"
+            }
           },
+          "diskSizeGB": 127,
+          "diskIOPSReadWrite": 500,
+          "diskMBpsReadWrite": 60,
           "encryption": {
             "type": "EncryptionAtRestWithPlatformKey"
           },
-          "timeCreated": "2016-12-28T04:41:35.079872+00:00",
-          "provisioningState": "Succeeded"
-        },
-        "type": "Microsoft.Compute/disks",
-        "location": "westus",
-        "tags": {
```

Comment:
tags are missing

Reply:
created the disk without tags, do I need to add tags?

Reply:
yes, its supposed to show all the possible info in get disk only new properties need to be added, existing ones shouldnt be removed

Reply:
oh I see okay, will update it.